src/mccann_hub/odoolib/connection/__init__.py

```
@@ -0,0 +1,2 @@
+from ._connection import Connection
```

Ruff (0.8.2)

`._connection.Connection` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)